src/js/content.ts

```diff
@@ -34,8 +34,8 @@ function gotMessage(message: any, sender: any, sendResponse: any) {
                 let coordinates = UtilsEngine.getOffset(elVideo);
                 divVideoHiighlight.style.top = coordinates.top + 'px';
                 divVideoHiighlight.style.left = coordinates.left + 'px';
-                divVideoHiighlight.style.width = coordinates.width + 'px';
-                divVideoHiighlight.style.height = coordinates.height + 'px';
+                divVideoHiighlight.style.width = coordinates.width - 10 + 'px';

```

Comment:
why - 10??

Reply:
Since the border width is 5px, we need to subtract 10 from the width to cover the exact video area